guides/doc-Release_Notes/topics/katello.adoc

```diff
@@ -4,14 +4,41 @@
 [id="katello-headline-features"]
 == Headline Features
 
-There are no highlights with Katello {KatelloVersion}.
+=== New All Hosts index page improvements ===
+On the experimental new All Hosts index page, you can now click 'Manage Columns' and add Katello-provided columns, including
+* RHEL Lifecycle status
```

Comment:
ACK from me once the whitespace in here is added (check out the rendered version.